storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -49,9 +52,21 @@ type encoder struct {
 	enc         encoding.Encoder
 }
 
-type encoderMap struct {
-	id       ts.ID
-	encoders map[time.Time][]encoder
+type encodersByTime struct {
+	id ts.ID
+	// int64 instead of time.Time because there is an optimized map access pattern
+	// for i64's
+	encoders map[int64][]encoder
```

Comment:
I'm currently creating a type alias for this called UnixNano to be used throughout the codebase wherever we would normally use a map[time.Time]. I'll replace this int64 with the appropriate type in a subsequent PR

Reply:
could you update this to use the new type

Reply:
yeah will do